pango/src/auto/font.rs

```diff
@@ -34,8 +33,8 @@ pub trait FontExt: 'static {
     #[doc(alias = "pango_font_describe_with_absolute_size")]
     fn describe_with_absolute_size(&self) -> Option<FontDescription>;
 
-    #[doc(alias = "pango_font_find_shaper")]
-    fn find_shaper(&self, language: &Language, ch: u32) -> Option<EngineShape>;
+    //#[doc(alias = "pango_font_find_shaper")]
+    //fn find_shaper(&self, language: &Language, ch: u32) -> /*Ignored*/Option<EngineShape>;
```

Comment:
Heh, how those functions were not deprecated as well

Reply:
You'd automatically get warnings if you used them (also in C), so maybe just laziness or was forgotten :)